### DISCUSSION TITLE
Mystery fish fish up elder guardian by any chance?

### DISCUSSION THREAD

**Discussion by Uboaaaaa** [Geolykt/EnchantmentsPlus#64] [General]
I'd like to know the fish 'loot table' of the enchantment 'Mystery fish'.
I remember some years ago getting killed by an elder guardian whilst afk fishing with that enchantment...

**Comment by Geolykt** [comment]
There is an equal chance to obtain squids, (elder-) guardian, cod, pufferfishes, salmon or tropical fishes. So yeah, there is still the chance to get this :slightly_smiling_face:
